api/oss/src/apis/fastapi/testsets/router.py

```diff
@@ -498,21 +518,64 @@ async def query_testsets(
             ):
                 raise FORBIDDEN_EXCEPTION  # type: ignore
 
-        testsets = await self.testsets_service.query_testsets(
-            project_id=UUID(request.state.project_id),
-            #
-            testset_query=testset_query_request.testset,
-            #
-            testset_refs=testset_query_request.testset_refs,
-            #
-            include_archived=testset_query_request.include_archived,
-            #
-            windowing=testset_query_request.windowing,
-        )
+        requested_refs = testset_query_request.testset_refs or []
```

Comment:
@ardaerzin 
Just to be sure about this change:
Is it doing more than using the `_infer...()` from above? It looks like there is a `testset_refs` that is also used.